client/sites-dashboard/components/sites-ellipsis-menu.tsx

```diff
@@ -176,6 +180,32 @@ const PreviewSiteModalItem = ( { recordTracks, site }: SitesMenuItemProps ) => {
 	);
 };
 
+const CopySiteItem = ( { recordTracks, site }: SitesMenuItemProps ) => {
+	const { __ } = useI18n();
+	const hasAtomicFeature = useSafeSiteHasFeature( site.ID, WPCOM_FEATURES_ATOMIC );
+	const userId = useSelector( ( state ) => getCurrentUserId( state ) );
+	const plan = site.plan;
+	const isSiteOwner = site.site_owner === userId;
+
+	if ( ! hasAtomicFeature || ! isSiteOwner || ! plan ) {
+		return null;
+	}
+
+	const copySiteHref = addQueryArgs( `/setup/copy-site`, {
+		productSlug: plan.product_slug,
```

Comment:
I wonder if we need some hardening around this... could a user manipulate this to assign a `personal` plan during the copy site process?

Reply:
If a user changes the URL, then they would buy a personal plan, and the copy site "migration" would fail in the backend. 
A user can always change the code in the client, so all the checks should be in the backend. But I agree that changing a URL is very tempting. Let's avoid passing the `productSlug` in the URL to make it harder to modify.

- I've created the issue: 1490-gh-Automattic/dotcom-forge